multi-arch-builders/provisioning/ppc64le/variables.tf

```diff
@@ -0,0 +1,78 @@
+
+variable "ibmcloud_api_key" {
+    description = "Denotes the IBM Cloud API key to use"
+    default = ""
+}
+
+variable "ibmcloud_region" {
+    description = "Denotes which IBM Cloud region to connect to"
+    default     = "us-south"
+}
+
+#INSERTED FOR MULTI-ZONE REGION SUCH AS FRANKFURT
+
+variable "ibmcloud_zone" {
+    description = "Denotes which IBM Cloud zone to connect to - .i.e: eu-de-1 eu-de-2  us-south etc."
+    default     = "us-south"
+}
+
+# Got the ID from `ibmcloud resource service-instances --long field` command, refer GUID for the instance
+variable "power_instance_id" {
+    description = "Power Virtual Server instance ID associated with your IBM Cloud account (note that this is NOT the API key)"
+    default = "556eb201-32bf-4ae2-8ab5-dfd7bbe97789"
+}
+
+variable "memory" {
+    description = "Amount of memory (GB) to be allocated to the VM"
+    default     = "50"
+}
+
+variable "processors" {
+    description = "Number of virtual processors to allocate to the VM"
+    default     = "15"
+}
```

Comment:
15 seems like a random number - would have expected a power of 2 (like 8 or 16) here.

Reply:
Each server model has a maximum allowable number, which may vary depending on utilization (interesting)

On the e880, the maximum core availability is now 12.04, reduced from the previous 15 I added here.
We now have on the e980, with  31.5. I will add a note in how to check it before ,so we may add the max we can.